ethereum/consensus-specs
# Release v1.5.0-alpha.9

- [x] testgen
- [x] version bump #3981
- [x] #3754
- [x] #3767
- [ ] ~~#3800~~
- [x] #3884
- [x] #3885
- [x] #3893
- [x] #3900
- [x] #3979
- [x] #3980
- [x] #3984
- [x] #3990
- [ ] ~~#3994~~
- [x] #3996
- [x] #3998
- [x] #4000
- [x] #4002
- [x] #4008 
- [ ] ~~#3553~~

<details>
<summary>work-in-progress release notes</summary>
<br>

`v1.5.0-alpha.9` -- Ampharos -- is the alpha release for the coming Electra upgrade.

_PR showing full diff can be found here: `#3982`_

## Electra

- Uncouple blob limits per block across CL and EL `#3800`
- Separate type for unaggregated network attestations `#3900`
- Update validator spec to handle execution requests `#3976`
- Fix validator fields in `get_validator_from_deposit` `#3978`
- Fix withdrawals bug and add more tests `#3979`
- Rename index vars in `get_attesting_indices` `#3996`
- Exclude empty requests in requests list `#3998`
- Add missed exit checks to consolidation processing `#4000`
- Ensure non-zero bits for each committee bitfield comprising an aggregate `#4002`

## EIP-7594 (PeerDAS)

- Rebase eip7594 onto Electra `#3893`
- Specify data column sidecar verification functions to run `#3963`
- Add `MAX_BLOBS_PER_BLOCK_EIP7594` and corresponding configs `#4008`

## Networking

- Deprecate `TTFB`, `RESP_TIMEOUT`, introduce rate limiting recommendations `#3767`

## Testing, repo, etc

- Encode zero-length hex-strings with quotes `#3754`
- Avoid generating empty transactions in tests `#3884`
- Synchronously check all transactions to have non-zero length `#3885`
- Fix blob transaction serialization to use RLP `#3904`
- Move deposit contract rules to its own `Makefile` `#3940`
- Add new `check_mods` function for generators `#3970`
- In `README`, move outdated specs to new section `#3973`
- Fix invalid file references in markdown files `#3975`
- Add misc beacon chain operations tests `#3980`
- Add more epoch processing tests `#3984`
- Add gen-modcheck CI check `#3986`
- Add more transition tests `#3990`
- Fix all `pytest` warnings `#3989`
- Upgrade `pylint` & split config files `#3991`
- Add test to check that pending deposits are sorted at fork `#3995`

</details>
